src/views/DepositWithdraw/components/DepositInfo/index.tsx

```diff
@@ -75,8 +75,8 @@ export const DepositInfo: React.FC<IDepositInfo> = ({ farmSelected }) => {
         <SmallInfoCard
           iconType={'question'}
           statTitle={'time left'}
-          statContent={`${timeRemaining?.days} days left`}
-          statFill={formatFillPercentage(timeRemaining, totalTime)}
+          statContent={`${formatDaysFromTimestamp(timeLeft).toFixed(0)} days left`}
```

Comment:
Is `toFixed(0)` necessary here? Won't `timeLeft` always return an integer?

Reply:
After `formatDaysFromTimestamp(timeLeft)` it returns days including lots of decimals. And it's not practical to have 45.454853945 days left.

Reply:
Ah, right, I mistakenly thought you were calling `toFixed(0)` on `timeLeft`, not `formatDaysFromTimestamp()`.

Maybe the function should just be changed to:
```
export function formatDaysFromTimestamp(totalTime?: number): number {
  if (totalTime && totalTime !== 0) {
    const durationDays = totalTime / 86400;
    return Math.round(durationDays);
  } else {
    return 0;
  }
}
```
... so that we don't have to append `toFixed(0)` every time we use it.

Reply:
Sure, that sounds way better